react/features/base/config/interfaceConfigWhitelist.js

```diff
@@ -45,6 +45,7 @@ export default [
     'SETTINGS_SECTIONS',
     'SHOW_CHROME_EXTENSION_BANNER',
     'SHOW_DEEP_LINKING_IMAGE',
+    'SHOW_DEEP_LINKING_LOGO',
```

Comment:
This does not make sense to be whitelisted.

Reply:
I guess I don't understand the whitelist. I just assumed the whitelist was saying "only these config values can be set" but I must be wrong. I'll just remove it from the whitelist if you want.

Reply:
Guess if I had read the comment at the top of the file I would have known more...

```
The interface config keys to whitelist, the keys that can be overridden.
```

:man_facepalming:

Reply:
Those that can be overridden from the URL or external API.